### PR TITLE
Analytics docs 020

### PR DESCRIPTION
**Pull request by LoekL**
Opening the docs PR for 020!

### CODE REVIEW

File: docs/content/services-packages/analytics-pipeline/local.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+0. Find [the API key for your Google project](https://console.cloud.google.com/apis/credentials) that you created in [step 3.1 of the Deploy section]({{urlRoot}}/content/services-packages/analytics-pipeline/deploy#31---store-your-secret), which you need to pass via the `key` parameter in the URL of your POST request: `{{your_analytics_api_key}}`.
+
+0. Replace `{{your_analytics_api_key}}` with your API key in the curl request below and submit it in your terminal:
+
+```sh
+curl --request POST --header "Content-Type:application/json" --data "{\"eventSource\":\"client\",\"eventClass\":\"docs\",\"eventType\":\"endpoint_docker_compose\",\"eventTimestamp\":1562599755,\"eventIndex\":6,\"sessionId\":\"f58179a375290599dde17f7c6d546d78\",\"versionId\":\"2.0.13\",\"eventEnvironment\":\"testing\",\"eventAttributes\":{\"playerId\": 12345678}}" "http://0.0.0.0:8080/v1/event?key={{your_analytics_api_key}}&analytics_environment=testing&event_category=cold&session_id=f58179a375290599dde17f7c6d546d78"
+```
+
+A successful response looks like:
```
Comment (rebekahmullen) [inline docs/content/services-packages/analytics-pipeline/local.md]: i REALLY like that we put what successful responses look like. nice work!

File: docs/content/services-packages/analytics-pipeline/usage.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# Analytics Pipeline: usage
+<%(TOC)%>
+
+All Online Services have been [instrumented](https://en.wikipedia.org/wiki/Instrumentation_(computer_programming)) out-of-the-box. This means that whenever you deploy the Analytics Pipeline, you automatically capture analytics events originating from these services and can query them with BigQuery in your Google project.
```
Comment (rebekahmullen) [inline docs/content/services-packages/analytics-pipeline/usage.md]: ++ to adding what "instrumentation" means :)

File: docs/content/get-started/setup.md [hunk 3]
```diff
@@ -28,23 +28,37 @@ There are also a few things you need to set up before getting started:
...
+For example, if a container is defined as having 3 replicas, Kubernetes starts up 3 instances of that container. If one of them falls over or is stopped, Kubernetes immediately starts up another one for you. There will be more information on Kubernetes later in this guide but you can find out more on the [Kubernetes website](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/).<%(/Expandable)%>
 
-> **Tip:** If you use Google Cloud Platform, install [Google Cloud SDK](https://cloud.google.com/sdk/) - useful to push built images up to your Google Cloud project.
+Kubernetes Engine isn't free, but you can sign up to the free trial if you need to. You'll know it's ready to go when you can see this dialog box:
+
+![]({{assetRoot}}img/get-started/create-k8s-cluster.png)
+
+Note that you can port these configurations to run on [Amazon AWS](https://aws.amazon.com/), [Microsoft Azure](https://azure.microsoft.com/en-us/), [Alibaba Cloud](https://www.alibabacloud.com/) or any cloud hosting service.
```
Comment (davej-improbable) [inline docs/content/get-started/setup.md]: Sugg: worth including DigitalOcean here too? Feels like a good middle-of-the-road alternative to the ones listed.

File: services/k8s/deployment-pool/deployment.yaml [hunk 4]
```diff
@@ -64,7 +64,7 @@ spec:
         - --minimum-ready-deployments
         - "3"
         - --match-type
-        - "{{your_match_type}}" # Update
+        - "{{your_match_type}}" # Update, to for instance "match" (which is the default setting of the sample-matcher)
```
Comment (rebekahmullen) [inline services/k8s/deployment-pool/deployment.yaml]: "to for instance match" ... i'm not sure the wording is right here

File: services/terraform/variables.tf [hunk 5]
```diff
@@ -11,7 +11,8 @@ variable "gcloud_region" {}
 # Pick one from: https://cloud.google.com/compute/docs/regions-zones/
 variable "gcloud_zone" {}
 
-# Bucket location, either US or EU.
-variable "gcloud_bucket_location" {}
-
+# The name of your Kubernetes cluster.
 variable "k8s_cluster_name" {}
+
+# Cloud Storage location, either US or EU.
```
Comment (davej-improbable) [inline services/terraform/variables.tf]: Put backticks around `US` and `EU` to indicate those are the literal acceptable values for this paramer.

File: docs/content/services-packages/gateway/local.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+# Gateway (including matchmaking): local
+<%(TOC)%>
+
+When you are developing your game in SpatialOS, you can run it locally on your development machine as if it were in the cloud. This is useful for fast development and testing iterations. You can also run Online Services locally using `docker-compose`. You use this tool to start up multiple containers and ensure they're on the same network.
+
+## Prerequisites
+
+* This page assumes you have already [deployed the Gateway]({{urlRoot}}/content/services-packages/gateway/deploy).
+* If you're on Windows, you need to take some additional steps to mount Docker volumes. These steps are in a separate [Docker volumes on Windows]({{urlRoot}}/content/workflows/docker-windows-volumes.md) guide.
+
```
Comment (davej-improbable) [inline docs/content/services-packages/gateway/local.md]: Maybe add a line here about the values in double-curly brackets. For instance `* During configuration you'll need to note down various values. We have labelled these with {{curly_brackets}}, and will refer back to them later on in this guide.`

File: docs/content/services-packages/gateway/usage.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+# Gateway (including matchmaking): usage
+<%(TOC)%>
+
+The full connection flow goes something like this:
+
+1. Talk to [PlayFab](https://api.playfab.com/docs/tutorials/landing-players/best-login) to get a token.
+2. Exchange the PlayFab token for a player identity token (PIT) via the PlayFab Auth system you deployed.
```
Comment (davej-improbable) [inline docs/content/services-packages/gateway/usage.md]: "PlayFab Auth service" rather than "PlayFab Auth system"?

File: docs/content/services-packages/deployment-pool/overview.md [hunk 8]
```diff
@@ -16,11 +16,11 @@ The Deployment Pool module maintains game deployments in a ready-to-go state. It
...
 
-| Action       | Purpose      |
-|--------------|--------------|
-| `create`     | Creates a new deployment for the pool. This is generated when there are fewer "ready-to-go" deployments than required. |
-| `delete`     | Shuts down a running deployment. This is generated when a deployment is marked as "completed" and so is no longer required. A deployment puts itself in this state. |
-| `update`     | Changes a deployment's metadata once a process has finished. This is usually generated when a deployment has finished starting up and can be transitioned to the "ready-to-go" state. |
+| Action | Purpose |
+|--------|---------|
+| `create` | Creates a new deployment for the pool. This is generated when there are fewer "ready-to-go" deployments than required. |
+| `delete` | Shuts down a running deployment. This is generated when a deployment is marked as "completed" and so is no longer required. A deployment puts itself in this state. |
```
Comment (davej-improbable) [inline docs/content/services-packages/deployment-pool/overview.md]: Suggestion: "A deployment will put itself in this state automatically."

File: docs/content/services-packages/deployment-pool/local.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
+# Deployment Pool: local
+<%(TOC)%>
+
+## Prerequisites
+
+* This page assumes you have already [deployed the Deployment Pool]({{urlRoot}}/content/services-packages/deployment-pool/deploy).
+* To run the Deployment Pool, you need a previously uploaded assembly, a snapshot file and a launch configuration file.
```
Comment (davej-improbable) [inline docs/content/services-packages/deployment-pool/local.md]: Are there any pages we can link to explaining assemblies, snapshots and launch configs?

File: docs/content/services-packages/deployment-pool/deploy.md [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+
+0. The tool you need to use is at [`github.com/spatialos/online-services/tree/master/tools/ServiceAccountCLI`](https://github.com/spatialos/online-services/tree/master/tools/ServiceAccountCLI). You can read more about it in the [Platform service-account CLI documentation]({{urlRoot}}/content/workflows/service-account-cli). Navigate to the `/tools/ServiceAccountCLI` directory and run the following command, replacing the `--project_name` parameter with the name of your SpatialOS project (you can change `--service_account_name` to whatever you want, but we've used "online_services_demo" as an example):
+
+```sh
+dotnet run -- create --project_name "{{your_spatialos_project_name}}" --service_account_name "online_services_demo" --refresh_token_output_file=service-account.txt --lifetime=0.0:0 --project_write
+```
+
+This sets the lifetime to `0.0:0` (in other words, 0 days, 0 hours, 0 minutes), which just means that the refresh token will never expire. You might want to set it to something more appropriate to your needs.
+
+0. Mount the secret you created in into Kubernetes:
```
Comment (davej-improbable) [inline docs/content/services-packages/deployment-pool/deploy.md]: There's an extra "in" here :)

File: docs/content/services-packages/deployment-pool/deploy.md [hunk 11]
```diff
@@ -0,0 +1,246 @@
...
+
+0. Under “API restrictions”, select “Restrict key” and then choose ”Analytics REST API”.
+
+0.Next, mount the API key into Kubernetes as a secret, replacing `{{your_analytics_api_key}}` with the API key you just created:
+
+```sh
+kubectl create secret generic "analytics-api-key" --from-literal="analytics-api-key={{your_analytics_api_key}}"
+```
+
+<%(Callout type="info" message="Note that you need to enter the actual key, not the path to it.")%>
```
Comment (davej-improbable) [inline docs/content/services-packages/deployment-pool/deploy.md]: Maybe rephrase to "Note that you need to enter the actual analytics API key here rather than the path to one."

File: docs/content/get-started/setup.md (hunk 3 above)
Comment (JoelOtter) [reply docs/content/get-started/setup.md]: +1! it's cheap too

Comment (LoekL) [reply docs/content/get-started/setup.md]: Maybe we should also add TencentCloud? They support k8s: https://www.terraform.io/docs/providers/tencentcloud/r/kubernetes_cluster.html

File: docs/content/services-packages/deployment-pool/local.md (hunk 9 above)
Comment (LoekL) [reply docs/content/services-packages/deployment-pool/local.md]: Will link to:

- https://docs.improbable.io/reference/latest/shared/glossary#assembly
- https://docs.improbable.io/reference/latest/shared/glossary#snapshot
- https://docs.improbable.io/reference/latest/shared/glossary#launch-configuration-file

File: docs/content/services-packages/analytics-pipeline/usage.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+## Using the endpoint
+
+To send more data to the Analytics Pipeline, you can make simple `POST` requests with a JSON payload to your endpoint. We recommend the payload to be a JSON list of approximately 100 events.
+
+The endpoint URL takes six parameters:
+
+| Parameter | Required/Optional | Description |
+|-----------|-------------------|-------------|
+| `key` | Required | Must be tied to your Google project ([info](https://cloud.google.com/endpoints/docs/openapi/get-started-kubernetes#create_an_api_key_and_set_an_environment_variable)). |
+| `analytics_environment` | Optional | The environment, for example {`testing`, `development` (default), `staging`, `production`}. |
```
Comment (davej-improbable) [inline docs/content/services-packages/analytics-pipeline/usage.md]: Not sure about the curlies here. Maybe change to "The environment, for example `testing`, `development` (the default), `staging`, or `production`."

File: docs/content/services-packages/deployment-pool/overview.md (hunk 8 above)
Comment (LoekL) [reply docs/content/services-packages/deployment-pool/overview.md]: Adopted suggestion

File: docs/content/services-packages/analytics-pipeline/usage.md [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+| `session_id` | Optional | Should be set, otherwise defaults to `session-id-not-available`. |
+
+These parameters (except for `key`) play a part in how the data ends up in the GCS bucket:
+
+> gs://[[your Google project id]]-analytics/data_type=[[data_type]]/analytics_environment={{analytics_environment}}/event_category={{event_category}}/event_ds={{event_ds}}/event_time={{event_time}}/{{session_id}}/[[timestamp]]-[[random_alphanum]]
+
+Note that the endpoint:
+
+* automatically determines `[[data_type]]`. It can either be `json` (when valid JSON is `POST`ed) or `unknown` (otherwise).
+* automatically sets the fields `[[timestamp]]` and `[[random_alphanum]]` (a random string to avoid collisions) as well.
```
Comment (davej-improbable) [inline docs/content/services-packages/analytics-pipeline/usage.md]: Suggest rewording to "Note that the endpoint will:" and updating the phrasing of the bulletpoints to match.

File: docs/content/services-packages/analytics-pipeline/usage.md [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+| `eventIndex` | integer | Increments by one with each event per `sessionId`. Allows you to spot missing data. |
+| `eventClass` | string | A higher order mnemonic classification of the event (for example `session`). |
+| `eventType` | string | A mnemonic event description (for example `session_start`). |
+| `playerId` | string | A player's unique identifier, if available. |
+| `eventTimestamp` | float | The timestamp of the event, in Unix time. |
+| `eventAttributes` | dict | You can capture anything else relating to this particular event in this attribute as a nested JSON dictionary. |
+
+Keep the following in mind:
+
+* keys should always be camelCase, whereas values should be snake_case whenever they are of the type `string`.
```
Comment (davej-improbable) [inline docs/content/services-packages/analytics-pipeline/usage.md]: Maybe encase `camelCase` and `snake_case` in backticks?

File: docs/content/services-packages/gateway/local.md (hunk 6 above)
Comment (LoekL) [reply docs/content/services-packages/gateway/local.md]: It shouldn't go there as it isn't a prerequisite.

I will add the following under l.11 (## Deploy locally):

In the beginning of this section you will need to note down a few values. We have labelled these `{{in_double_curly_brackets}}`, and will refer back to them in later the section.

File: docs/content/get-started/setup.md (hunk 3 above)
Comment (JoelOtter) [reply docs/content/get-started/setup.md]: Perhaps it would be useful (if, possibly, overkill) to chat to product marketing and ask which cloud providers they'd like us to shout out. I'd guess they'd want us to mention tencent in light of the new deal.

File: services/k8s/deployment-pool/deployment.yaml (hunk 4 above)
Comment (JoelOtter) [reply services/k8s/deployment-pool/deployment.yaml]: Maybe `Update to, for instance, "match"` ?

File: docs/content/services-packages/analytics-pipeline/usage.md [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+* you should nest anything that is custom to a particular event within `eventAttributes`.
+
+## Example POST request
+
+A `POST` request to the analytics endpoint that includes invoking the analytics Cloud Function (`event_category=function`) looks like this:
+
+```sh
+curl --request POST --header "Content-Type:application/json" --data "[{\"eventEnvironment\":\"testing\",\"eventSource\":\"client\",\"sessionId\":\"f58179a375290599dde17f7c6d546d78\",\"versionId\":\"0.0.1\",\"eventIndex\":0,\"eventClass\":\"docs\",\"eventType\":\"test\",\"playerId\":\"12345678\",\"eventTimestamp\":1562599755,\"eventAttributes\":{\"hello\":\"world\"}},{\"eventEnvironment\":\"testing\",\"eventSource\":\"client\",\"sessionId\":\"f58179a375290599dde17f7c6d546d78\",\"versionId\":\"0.0.1\",\"eventIndex\":1,\"eventClass\":\"docs\",\"eventType\":\"test\",\"playerId\":\"12345678\",\"eventTimestamp\":1562599755,\"eventAttributes\":{\"hello\":\"world\"}}]" "http://analytics.endpoints.{{your_google_project_id}}.cloud.goog:80/v1/event?key={{your_analytics_api_key}}&analytics_environment={{analytics_environment}}&event_category={{event_category}}&session_id={{session_id}}"
```
Comment (davej-improbable) [inline docs/content/services-packages/analytics-pipeline/usage.md]: Such a shame Windows doesn't permit the use of single-quotes here to make it a bit more pleasant! Could we maybe provide the JSON separately here, ask them to save it to a file (e.g. `hello_world.json`) and then the command could be:

```sh
curl --header "Content-Type:application/json" --data @hello_world.json "http://analytics.endpoints.{{your_google_project_id}}.cloud.goog:80/v1/event?key={{your_analytics_api_key}}&analytics_environment={{analytics_environment}}&event_category={{event_category}}&session_id={{session_id}}"
```

File: docs/content/services-packages/gateway/local.md (hunk 6 above)
Comment (LoekL) [reply docs/content/services-packages/gateway/local.md]: I added the same sentence in the local execution pages of the deployment pool & analytics pipeline.

File: docs/content/services-packages/analytics-pipeline/overview.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
+# Analytics Pipeline: overview
+<%(TOC)%>
+
+You can use the Analytics Pipeline to capture and persist JSON analytics events that come from other Online Services, games, or any other type of source that is connected to the internet. Unlike the other Online Services, which use C#, the Analytics Pipeline uses Python, because it is highly popular among data professionals, and because it meant that we could use a single language across all elements of the pipeline. The Analytics Pipeline is currently only supported on Google Cloud.
```
Comment (davej-improbable) [inline docs/content/services-packages/analytics-pipeline/overview.md]: Could we call out the specific features of Google Cloud that are being used that prevent it being hosted on other providers?

File: docs/content/services-packages/gateway/usage.md (hunk 7 above)
Comment (LoekL) [reply docs/content/services-packages/gateway/usage.md]: Changed to system!

File: docs/content/services-packages/analytics-pipeline/local.md [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+```sh
+docker-compose -f services/docker/docker_compose_local_analytics_pipeline.yml up
+```
+
+0. Find [the API key for your Google project](https://console.cloud.google.com/apis/credentials) that you created in [step 3.1 of the Deploy section]({{urlRoot}}/content/services-packages/analytics-pipeline/deploy#31---store-your-secret), which you need to pass via the `key` parameter in the URL of your POST request: `{{your_analytics_api_key}}`.
+
+0. Replace `{{your_analytics_api_key}}` with your API key in the curl request below and submit it in your terminal:
+
+```sh
+curl --request POST --header "Content-Type:application/json" --data "{\"eventSource\":\"client\",\"eventClass\":\"docs\",\"eventType\":\"endpoint_docker_compose\",\"eventTimestamp\":1562599755,\"eventIndex\":6,\"sessionId\":\"f58179a375290599dde17f7c6d546d78\",\"versionId\":\"2.0.13\",\"eventEnvironment\":\"testing\",\"eventAttributes\":{\"playerId\": 12345678}}" "http://0.0.0.0:8080/v1/event?key={{your_analytics_api_key}}&analytics_environment=testing&event_category=cold&session_id=f58179a375290599dde17f7c6d546d78"
```
Comment (davej-improbable) [inline docs/content/services-packages/analytics-pipeline/local.md]: Can make this a bit shorter by dropping `--request POST` as cURL will imply it automatically from the `--data` parameter.

File: services/k8s/deployment-pool/deployment.yaml (hunk 4 above)
Comment (LoekL) [reply services/k8s/deployment-pool/deployment.yaml]: What about when I use back ticks: `# Update, to for instance `match` (which is the default setting of the sample-matcher)`

The suggestion is to set to `match`. Not that it whatever you set it to "should match X".

File: services/terraform/variables.tf (hunk 5 above)
Comment (LoekL) [reply services/terraform/variables.tf]: Updated to: ```# BigQuery & Cloud Storage location, either `US` or `EU`, see: https://cloud.google.com/bigquery/docs/locations#multi-regional_locations```

File: services/k8s/deployment-pool/deployment.yaml (hunk 4 above)
Comment (LoekL) [reply services/k8s/deployment-pool/deployment.yaml]: Ah, missed your comment @JoelOtter ! Will use: ```Update to, for instance, `match` ```

File: docs/content/services-packages/deployment-pool/deploy.md (hunk 10 above)
Comment (LoekL) [reply docs/content/services-packages/deployment-pool/deploy.md]: Same mistake was present in a few other areas as well, fixed all of them!

File: docs/content/services-packages/analytics-pipeline/local.md (hunk 17 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/local.md]: I'd rather leave it explicit as it is, I for instance didn't know it would be implied by curl if I'd left it out, and would question whether it's `PUT` or `POST`, etc.

File: docs/content/get-started/setup.md (hunk 3 above)
Comment (LoekL) [reply docs/content/get-started/setup.md]: I'll check with them

File: docs/content/services-packages/deployment-pool/deploy.md (hunk 11 above)
Comment (LoekL) [reply docs/content/services-packages/deployment-pool/deploy.md]: Changed to: Note that you need to enter the actual *analytics API* key, not the path to it.

File: docs/content/services-packages/analytics-pipeline/usage.md (hunk 12 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/usage.md]: I was thinking along the lines of a [set](https://en.wikipedia.org/wiki/Set_(mathematics)) of options. WDYT?

File: docs/content/services-packages/analytics-pipeline/usage.md (hunk 13 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/usage.md]: Adopted

File: docs/content/services-packages/analytics-pipeline/usage.md (hunk 14 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/usage.md]: Adopted

File: docs/content/services-packages/analytics-pipeline/usage.md (hunk 12 above)
Comment (davej-improbable) [reply docs/content/services-packages/analytics-pipeline/usage.md]: My concern is that introducing curly brackets here could possible confuse those who aren't familiar with formal set notation or might think it's related to our `{{double-curly}}` notation elsewhere, whereas plain English feels a bit safer.

File: docs/content/services-packages/analytics-pipeline/local.md (hunk 17 above)
Comment (davej-improbable) [reply docs/content/services-packages/analytics-pipeline/local.md]: Fair enough :)

File: docs/content/services-packages/analytics-pipeline/overview.md (hunk 16 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/overview.md]: I'd say it's so much it doesn't make sense to call out. 

I use Cloud Endpoints - which afaik doesn't generalize nicely across cloud providers. BigQuery is GCP specific. My Dataflow/Apache Beam script is using GCP connectors, etc.

Comment (davej-improbable) [reply docs/content/services-packages/analytics-pipeline/overview.md]: Hmm, in that case maybe remove the "currently" part of the sentence, as that implies support for other providers is on the way and it sounds like this might not be the case!

File: docs/content/services-packages/analytics-pipeline/usage.md (hunk 15 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/usage.md]: How about we show your suggestion by default, but add a breakout box below with the original `curl` statement, to show them how it's done when passing the JSON as a string? 

Mainly because the string way is likely how their instrumentation will operate.

Comment (davej-improbable) [reply docs/content/services-packages/analytics-pipeline/usage.md]: I like that!

File: docs/content/services-packages/analytics-pipeline/overview.md (hunk 16 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/overview.md]: Will remove **currently**. 

I am wondering whether we/I should work on providing an AWS version at some point though.

File: docs/content/services-packages/analytics-pipeline/usage.md (hunk 12 above)
Comment (LoekL) [reply docs/content/services-packages/analytics-pipeline/usage.md]: Changed to plain English!

File: docs/content/get-started/setup.md (hunk 3 above)
Comment (LoekL) [reply docs/content/get-started/setup.md]: For now I'm going with: Note that you can port these configurations to run on [Amazon Web Services](https://aws.amazon.com/), [Microsoft Azure](https://azure.microsoft.com/en-us/), [Alibaba Cloud](https://www.alibabacloud.com/), [DigitalOcean](https://www.digitalocean.com/), [Tencent Cloud](https://intl.cloud.tencent.com/) or any cloud hosting service.

Will tweak if they get back to me and suggest to change.

Comment (davej-improbable) [reply docs/content/get-started/setup.md]: Suggest adding "...that supports Kubernetes" to the end :)

**Comment by anne-edwards** [comment]
Just fixed some numbering. Note that this is a hack (and there are inconsistencies/it doesn't look great) because of Markdown issues that occur when you use code blocks within numbered lists, but the "fix" at least means that the numbers go in the right order.

File: services/terraform/module-playfab-auth/endpoints.tf [hunk 18]
```diff
@@ -0,0 +1,20 @@
+# This file creates the endpoints required for the Gateway.
```
Comment (JoelOtter) [inline services/terraform/module-playfab-auth/endpoints.tf]: nit: this comment should probably be changed
